src/display_servers/xlib_display_server/xwrap.rs

```diff
@@ -1252,6 +1252,21 @@ impl XWrap {
         }
     }
 
+    pub fn unfocus(&self) {
+        let windows = self.get_windows_for_root(self.get_default_root());
+        if let Ok(windows) = windows {
+            let dock = windows.iter().find(|w| {
+                self.get_atom_prop_value(**w, self.atoms.NetWMWindowType)
+                    == Some(self.atoms.NetWMWindowTypeDock)
+            });
+            if let Some(dock) = dock {
+                self.window_take_focus(&Window::new(WindowHandle::XlibHandle(*dock), None, None));
+                return;
+            }
+        }
+        self.window_take_focus(&Window::new(self.get_default_root_handle(), None, None));
+    }
+
```

Comment:
I say rather than using window_take_focus to use ```self.send_xevent_atom(handle, self.atoms.WMTakeFocus);```,
as for the dock most of the function is skipped (plus we don't care whether the user clicks it) and maybe the root window? (idk if the root window does anything if you ask to take focus, and if it does it might be less buggy then us forcing the focus 🤷‍♂️).